meerk40t/core/elements.py

```diff
@@ -1696,11 +1752,12 @@ def operation_list(channel, _, data=None, **kwgs):
                         channel(name)
             channel(_("----------"))
 
+        @context.console_option("color", "c", type=Color)
+        @context.console_option("default", "d", type=bool)
         @context.console_option("speed", "s", type=float)
         @context.console_option("power", "p", type=float)
         @context.console_option("step", "S", type=int)
         @context.console_option("overscan", "o", type=Length)
-        @context.console_option("color", "c", type=Color)
         @context.console_option("passes", "x", type=int)
         @context.console_command(
             ("cut", "engrave", "raster", "imageop", "dots"),
```

Comment:
The default is never listed in the `makeop()` def. It needs a line added there to access the -d option. Also, I don't know of bool there would need action="store_true" else it might require a value.

Reply:
Also the `@context.console_option("default", "d", type=bool, action='store_true')` is strongly preferred for the default line. I don't think it deals with the bools that well without it since it requires they take a value.

Reply:
I fixed the missing parameter. I have no idea whether bool will work or not.